.github/workflows/satis-webhook.yml

```diff
@@ -26,7 +26,7 @@ jobs:
         with:
           token: ${{ secrets.WEBHOOK_TOKEN }}
           repository: bluehost/satis
-          event-type: 'Trigger Satis Build'
+          event-type: 'satis-build'
```

Comment:
Since this is hosted on the Bluehost org, the event type shouldn't change since "Trigger Satis Build" is what the Bluehost Satis is listening for.

Reply:
Good catch, thanks!